ui/msg/src/view/search.ts

```diff
@@ -30,15 +30,15 @@ export function renderInput(ctrl: MsgCtrl): VNode {
 export function renderResults(ctrl: MsgCtrl, res: SearchResult): VNode {
   return h('div.msg-app__search.msg-app__side__content', [
     res.contacts[0] && h('section', [
-      h('h2', 'Discussions'),
+      h('h2', ctrl.trans.noarg('discussions')),
       h('div.msg-app__search__contacts', res.contacts.map(t => renderContacts(ctrl, t)))
     ]),
     res.friends[0] && h('section', [
-      h('h2', 'Friends'),
+      h('h2', ctrl.trans.noarg('friends')),
       h('div.msg-app__search__users', res.friends.map(u => renderUser(ctrl, u)))
     ]),
     res.users[0] && h('section', [
-      h('h2', 'Players'),
+      h('h2', ctrl.trans.noarg('players')),
```

Comment:
This won't translate for some reason. If I substitute in `ctrl.trans.noarg('today')`, it _does_ translate. Any ideas?

Reply:
only selected translations are sent to the client for performance reasons https://github.com/ornicar/lila/blob/ada5aa68a5e00a7610ef4a879c2f6c36a118feb8/app/views/msg.scala#L33-L41

Reply:
Yeah, I have passed `players` there: https://github.com/ornicar/lila/pull/5990/files#diff-7328165f707f1b86e8c655bf1a078474R41-R46

`discussions` and `friends` do translate, but not players 😖

Reply:
I can make a new `val` called `players2` and pass it and it will translate too 🤷‍♂

Reply:
you can't touch that file, it's generated: https://github.com/ornicar/lila/blob/49864efc9d74564b38e338e7bea44431494a0797/modules/i18n/src/main/I18nKeys.scala#L1